gfx/NeoDays/pngs_tiles_10x10/furniture/plant/f_mutpoppy_season_winter_f_dandelion_season_winter_f_flower_spurge_season_winter_f_black_eyed_susan_season_winter_f_sunflower_season_winter_f_datura_s.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Should probably shorten the name of the file. The super long name of json were automatically generated by the decompose.py script and that sometimes causes issue with github because the names are too long.